js/index.js

```diff
@@ -0,0 +1,72 @@
+
+/* call the container that we will put in it the items and submit button */
+const app = document.getElementById('anime-container');
+const bt = document.getElementById('search-btn');
+bt.addEventListener('click',subbtn);
+
+/* when we click the button it will call the function */
+/* this func fetch the api and check if it have correct connection */
+function subbtn(){
+    var val = document.getElementById('search');
+    var y = val.value;
+    console.log(y);
+fetch(`https://api.jikan.moe/v3/search/anime?q=${y}`)
+.then((response) => {
+    if (response.status !== 200){
+    console.log(`looks like there is a problem . status code : ${response.status}`)
+} else {
+    return response.json()
+}})
+
+
+/* if the api have correct conection this will bring the value in search input and look for it in api then it will create the elements down */
+.then((data) => {
+    console.log(data.results)
```

Comment:
delete this line there is no need for it.

Reply:
line 24

Reply:
ok